blocks/themes/uc-basic/config.css

```diff
@@ -13,7 +13,7 @@
   --cfg-thumb-size: 76;
   --cfg-show-empty-list: 0;
   --cfg-use-local-image-editor: 0;
-  --cfg-use-cloud-image-editor: 1;
+  --cfg-use-cloud-image-editor: 0;
```

Comment:
Why?

Reply:
Cloud editor was broken that time, i've switched it back.